public/js/app.js

```diff
@@ -0,0 +1,315 @@
+                } else if (email === "") {
+                    msg = $('<div class="echec">Vous devez saisir votre mail</div>');
+                } else if (message === "") {
+                    msg = $('<div class="echec">Vous devez saisir un message</div>');
+                } else {
+                    $.ajax({
+                        type: "POST",
+                        url: $(this).attr("action"),
+                        data: $(this).serialize(),
+                        success: function () {
+
+                            var msgSuccess;
+                            msgSuccess = $('<div class="merci">Merci ' + nom + ', votre message a bien été envoyé </div>');
+                            $("body").append(msgSuccess);
+                            setTimeout(function () {
+                                    $(msgSuccess).fadeOut();
+                                },
+                                2300
+                            );
+
+                            document.getElementById("nom").value = "";
+                            document.getElementById("email").value = "";
+                            document.getElementById("message").value = "";
+                        },
+                        error: function () {
+                            var msgError;
+                            msgError = $('<div class="echec">Le message n\'a pas été envoyé </div>');
+                            $("body").append(msgError);
+                            setTimeout(function () {
+                                    $(msgError).fadeOut();
+                                },
+                                2300
+                            );
+                        }
+                    });
+                }
+                $("body").append(msg);
+
+                setTimeout(function () {
+                        $(msg).fadeOut();
+                    },
+                    2300
+                );
+                return false;
+            });
+
+            /********** Centre d'interet change **********/
+
+            var TxtType = function (el, toRotate, period) {
+                this.toRotate = toRotate;
+                this.el = el;
+                this.loopNum = 0;
+                this.period = parseInt(period, 10) || 800;
+                this.txt = '';
+                this.tick();
+                this.isDeleting = false;
+            };
+
+            TxtType.prototype.tick = function () {
+                var i = this.loopNum % this.toRotate.length;
+                var fullTxt = this.toRotate[i];
+
+                if (this.isDeleting) {
+                    this.txt = fullTxt.substring(0, this.txt.length - 1);
+                } else {
+                    this.txt = fullTxt.substring(0, this.txt.length + 1);
+                }
+
+                this.el.innerHTML = '<span class="wrap">j\'\aime ' + this.txt + '</span>';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/vanmarcke/ocr_projet5_blog/pullRequest?prid=8116981)
```suggestion
                this.el.innerHTML = "<span class=\"wrap\">j'\aime " + this.txt + '</span>';
```